packages/types/src/base.ts

```diff
@@ -8,32 +8,14 @@ export type BIP44Params = {
   index?: number
 }
 
-export enum ChainTypes {
-  Ethereum = 'ethereum',
-  Bitcoin = 'bitcoin',
-  Cosmos = 'cosmos',
-  Osmosis = 'osmosis'
-}
-
-const supportedChainIds = [
-  'eip155:1',
-  'bip122:000000000019d6689c085ae165831e93',
-  'cosmos:cosmoshub-4',
-  'cosmos:osmosis-1'
-] as const
-
-export type SupportedChainIds = typeof supportedChainIds[number]
+export const SUPPORTED_CHAIN_IDS = {
+  EthereumMainnet: 'eip155:1',
+  BitcoinMainnet: 'bip122:000000000019d6689c085ae165831e93',
+  CosmosMainnet: 'cosmos:cosmoshub-4',
+  OsmosisMainnet: 'cosmos:osmosis-1'
```

Comment:
Something I have been thinking about with naming related to cosmos assets is including the version number in the key name (ie. `CosmosMainnet4`) to differentiate between mainnet chain upgrades.